app/users/edit/responsibilities/controller.js

```diff
@@ -7,11 +8,17 @@ import { all } from "rsvp";
 const UsersEditResponsibilitiesQueryParams = new QueryParams({});
 
 export default Controller.extend(UsersEditResponsibilitiesQueryParams.Mixin, {
+  router: service("router"),
+
   setup() {
     this.projects.perform();
     this.supervisees.perform();
   },
 
+  openSuperviseProfile: task(function* (superviseId) {
+    yield this.router.transitionTo("users.edit", superviseId);
+  }),
+
```

Comment:
It should be supervis**or** I guess? ~~openSuperviseProfile~~ -> `openSupervisorProfile`.

Further, why using tasks instead of actions here?